src/site/content/en/progressive-web-apps/pwa-checklist/index.md

```diff
@@ -0,0 +1,482 @@
+---
+layout: post
+title: What makes a good Progressive Web App?
+authors:
+  - samrichard
+  - petelepage
+date: 2020-01-06
+updated: 2020-01-06
+description: |
+  What makes a good, or great Progressive Web App?
+tags:
+  - post
+  - progressive-web-apps
+draft: true
+---
+
+<!-- Disable heading-increment because it mucks with the Details widget -->
+<!--lint disable heading-increment-->
+
+Progressive Web Apps (PWA) are built and enhanced with modern APIs to deliver
+native-like capabilities, reliability, and installability while reaching
+_anyone, anywhere, on any device_ with a single codebase. To help you create
+the best possible experience, use the [core](#core) and [optimal](#optimal)
+checklists and recommendations to guide you.
+
+## Core Progressive Web App checklist {: #core }
+
+Because PWAs span all devices, from mobile through desktop, the core
+Progressive Web App Checklist is all about what you need to do to make your
+app installable and usable by all users, regardless of screen size or input type.
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Starts fast, stays fast
+
+Performance plays a significant role in the success of any online experience,
+as high performing sites engage and retain users better than poorly performing
+ones. Sites should focus on optimizing for user-centric performance metrics.
+
+{% endDetailsSummary %}
+
+Performance plays a significant role in the success of any online experience,
+as high performing sites engage and retain users better than poorly performing
+ones. Sites should focus on optimizing for user-centric performance metrics.
+
+#### Why
+
+Speed is critical for actually getting users to _use_ your experience.
+In fact, as page load times go from 1 second to 10 seconds, the probability of
+a user bouncing increases by 123%. Performance doesn21ecac977bcat stop `onload`.
```

Comment:
Something got mangled after 'Performance'. 

'`onload`' --> 'with the `load` event.' To be grammatically correct, this sentence needs a prepositional phrase. `onload` is not a prepositional phrase even though it looks like one.